packages/magento-category/queries/GetCategoryStaticPaths.graphql

```diff
@@ -12,6 +12,14 @@ query GetCategoryStaticPaths($rootCategory: String!) {
           children {
             uid
             url_path
+            children {
+              uid
+              url_path
+              children {
+                uid
+                url_path
+              }
+            }
```

Comment:
Wellicht is dit ook nog een query welke we zouden kunnen gebruiken:

https://graphcommerce-mesh.fly.dev/graphql?query=query+%7B%0A++categories%28filters%3A+%7B%7D%2C+pageSize%3A+300%29+%7B%0A++++items+%7B%0A++++++name%0A++++++url_path%0A++++%7D%0A++%7D%0A%7D